### PR TITLE
renamed directives to assignments

### PR DESCRIPTION
**Pull request by calebjclark**
- Assignment is now consistently used throughout the code base as a replacement for the prior "directive".
- Also, I moved two files from runner/lib into docs/lib that were only being used by docs, and...
- browsersToTest.json has its `tests` array renamed to `agents`

### CODE REVIEW

File: REMOTE [hunk 1]
```diff
@@ -0,0 +1 @@
+174.138.36.46
```
Comment (blakebyrnes) [inline REMOTE:1]: I think we should probably have this in a .env file that doesn't get checked in

File: profiler/lib/runInstructionInWebDriver.ts [hunk 2]
```diff
@@ -1,17 +1,17 @@
...
-import IDirective from '@double-agent/runner/interfaces/IDirective';
-import IDirectivePage from '@double-agent/runner/interfaces/IDirectivePage';
+import IInstruction from '@double-agent/runner/interfaces/IInstruction';
+import IInstructionPage from '@double-agent/runner/interfaces/IInstructionPage';
 
-export default async function runDirectiveInWebDriver(
+export default async function runInstructionInWebDriver(
   driver: WebDriver,
-  directive: IDirective,
+  instruction: IInstruction,
```
Comment (blakebyrnes) [inline profiler/lib/runInstructionInWebDriver.ts:7]: The naming turns out to be a bit weird when it's a set of "instructions" per instruction, doesn't it? I'm now a bit thrown by the plural vs singular of this.

File: REMOTE (hunk 1 above)
Comment (calebjclark) [reply REMOTE]: Yep. I accidentally checked in this tmp file. Just did a force push to remove it.

File: profiler/lib/runInstructionInWebDriver.ts (hunk 2 above)
Comment (blakebyrnes) [reply profiler/lib/runInstructionInWebDriver.ts:7]: BTW, I know this is one of those "I'm not sure what to do with this" comments. If you feel like this works, I'll see if it grows on me. Just realized as I was reading the code that this was tripping me up in a different way.

Comment (calebjclark) [reply profiler/lib/runInstructionInWebDriver.ts:7]: When you say, " a set of instructions per instruction", are you referring to the pages array within the instruction?

Comment (calebjclark) [reply profiler/lib/runInstructionInWebDriver.ts]: `Assignment` seems to work better. PR has been updated.

**Pull request merged**